source/elements/oneDPL/source/parallel_api/execution_policies.rst

```diff
@@ -180,5 +178,54 @@ as the template argument, otherwise unspecified.
 Return a policy object constructed from ``policy``, with a new kernel name provided as the template
 argument. If no policy object is provided, the new policy is constructed from ``dpcpp_default``.
 
+.. _exec-policy-type-trait:
+
+Execution Policy Type Trait
++++++++++++++++++++++++++++
+
+oneDPL provides type trait utilities to detect its execution policy types at compile time for the purpose of
+function overload resolution:
+
+.. code:: cpp
+
+  // Defined in <oneapi/dpl/execution>
+
+  namespace oneapi {
+    namespace dpl {
+
+      template <class T>
+      struct is_execution_policy { /*see below*/ };
+
+      template <class T>
+      constexpr bool is_execution_policy_v = oneapi::dpl::is_execution_policy<T>::value;
+
+      namespace execution {
+
+        template <class T>
+        struct is_execution_policy { /*see below*/ };
+
+        template <class T>
+        constexpr bool is_execution_policy_v = oneapi::dpl::execution::is_execution_policy<T>::value;
+
+      }
+    }
+  }
+
+``oneapi::dpl::is_execution_policy`` and ``oneapi::dpl::execution::is_execution_policy`` must be treated as name aliases
+to the same class template. It is unspecified in which namespace the underlying class template and its specializations
+are defined.
```

Comment:
```suggestion
to the same class template. It is unspecified, which namespace the underlying class template and its specializations
are defined in.
```

Reply:
Spellcheckers do not complain :), so I will keep this intact.

Reply:
Sure, this is small suggestion. As far as I understand "_preposition_ + which" is consider to be poorer English. More common use is "bla-bla, _which_ <some words> + <_proposition_ at the end>. Again, not proposing to change that but for my curiosity we could summon a native speaker. @danhoeflinger, could you please share your opinion?

Reply:
I searched for clarification on this, and I have found that "preposition + which" is considered not poorer but more formal English.

Reply:
I agree that "in which" is more formal (but a bit awkward), and "which ... in" is less formal, and a "dangling preposition" https://www.thefreedictionary.com/Dangling-Prepositions.htm which is a classic English grammar issue.

I was trying to rephrase to avoid the problem, but I think it may make it less clear.

"The location of the underlying class template definition and its specializations is left unspecified."

Reply:
Thanks Dan, I like the last suggestion, will apply.

Reply:
> "It is unspecified, which namespace contains the underlying class template definition and its specializations."

I also like that.

> ... not poorer but more formal English.

Thanks for clarification